### PR TITLE
Alternative layout for installation types

### PR DESCRIPTION
**Pull request by ludeeus**
Alternative to https://github.com/home-assistant/analytics.home-assistant.io/pull/32

### CODE REVIEW

File: site/src/analytics-average.ts [hunk 1]
```diff
@@ -36,11 +36,12 @@ export class AnalyticsAverage extends LitElement {
   static styles = css`
     .grid {
       display: grid;
-      grid-template-columns: repeat(2, 1fr);
-      grid-template-rows: repeat(2, 1fr);
+      grid-template-columns: repeat(4, 1fr);
```
Comment (bramkragten) [inline site/src/analytics-average.ts:39]: Add a minmax here, so it does nice responsive.

Comment (bramkragten) [reply site/src/analytics-average.ts:39]: mmmm that might not work well here, then you will get a row of 3 and 1 at some breakpoints.

Then just add an extra breakpoint to get to 2 columns?

**Review by bramkragten — APPROVED** [review]

File: site/src/analytics-installation-types.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            textStyle: {
+              color: isDarkMode ? "#e1e1e1" : "#212121",
+            },
+          },
+        }}
+        .rows=${[
+          ["Operating System", lastEntry.installation_types.os],
+          ["Container", lastEntry.installation_types.container],
+          ["Core", lastEntry.installation_types.core],
+          ["Supervised", lastEntry.installation_types.supervised],
```
Comment (bramkragten) [inline site/src/analytics-installation-types.ts:46]: Let's move these around, or sort it by size? (order should still match the graph above)

File: site/src/analytics-versions.ts [hunk 3]
```diff
@@ -44,7 +44,7 @@ export class AnalyticsVersions extends LitElement {
         ]}
         .options=${{
           title: "Top 5 used versions",
-          chartArea: { width: "70%", height: "80%" },
+          chartArea: { width: "70%", height: "70%" },
```
Comment (bramkragten) [inline site/src/analytics-versions.ts:47]: Why do we have to make the height smaller? On mobile the chart is getting really small now?

Comment (ludeeus) [reply site/src/analytics-versions.ts:47]: Looked cleaner, can probably do 100% for mobile

**Pull request merged**